plugins/mcreator-localization/help/default/block/tick_rate.md

```diff
@@ -4,7 +4,6 @@ has no effect.
 Keep in mind that usually, blocks naturally generated will not tick by default, unless tick randomly
 is used.
 
-Default tick rate of 10 means the block will tick each 10 world ticks. There are 20 world ticks in
-a second so this means the block will tick twice a second.
+Default tick rate of 0 means the block will not tick automatically.
 
-If this parameter is set to 0, the block won't tick automatically.
\ No newline at end of file
+If this parameter is set to a number greater than 0, the block will tick automatically. There are 20 world ticks in a second.
```

Comment:
Change the French tooltip for this:
```
Ce paramètre contrôle la fréquence à laquelle le bloc est tick. Si Tick au hasard est sélectionné, ce paramètre n'a aucun effet.

Gardez à l'esprit qu'en général, les blocs générés naturellement ne tick pas par défaut, sauf si Tick au hasard est utilisé.

Le taux de ticks par défaut de 0 signifie que le bloc ne tickera pas automatiquement.

Si ce paramètre est plus grand que 0, le bloc tickera automatiquement. Il y a 20 ticks mondiaux dans une seconde. Cela signifie que le bloc tickera deux fois par seconde.
```

Reply:
Done https://github.com/MCreator/MCreator/pull/2968/commits/94f41fa737184ba057f34b17b559985ea8363ab1